libs/langchain/langchain/embeddings/mlflow.py

```diff
@@ -1,3 +1,6 @@
-from langchain_community.embeddings.mlflow import MlflowEmbeddings
+from langchain_community.embeddings.mlflow import (
+    MlflowCohereEmbeddings,
+    MlflowEmbeddings,
+)
 
-__all__ = ["MlflowEmbeddings"]
+__all__ = ["MlflowEmbeddings", "MlflowCohereEmbeddings"]
```

Comment:
should not update code in `langchain` package

Reply:
Fixed, only updating `community`